Homepage/homepage.css

```diff
@@ -0,0 +1,221 @@
+/* GENERIC BODY STYLING */
+
+:root {
+    --body-bg: #FFFF00;
+    --anchor-hover-color: #0a0a0a;
+    --code-example-bg: #282c34;
+
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: var(--body-bg);
+    color: #0a0a0a;
+    font-family: 'Nunito', sans-serif;
+}
+
+
+/* STYLING FOR THE HEADER.
+
+    EVERY PAGE THAT HAS A HEADER WILL DERIVE IT'S STYLING FROM THIS STYLESHEET 
+    SINCE THE HEADER IS COMMON TO EVERY PAGE.
+    
+    ALTHOUGH THE HEADER IN THE WELCOME PAGE IS THE SAME, THE STYLING FOR THAT 
+    IS EXPLICITLY WRITTEN IN styles.css SINCE IT IS UNNECESSARY TO REFERENCE THIS 
+    ENTIRE STYLESHEET JUST FOR THE WELCOME PAGE. for eg. THIS PAGE HAS STYLES FOR
+    THE HOMEPAGE SIDEBAR WHICH THE WELCOME PAGE DOES NOT NEED. */
+
+
+.header {
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    align-content: center;
+    margin-bottom: 3rem;
+    height: max-content;
+}
+
+.header-logo {
+    font-family: 'Gochi Hand', cursive;
```

Comment:
Font family , define in main class and use it as css variables

Reply:
Changes done and pushed. Thanks.